frontend/pages/hosts/details/cards/Activity/ActivityItems/InstalledCertificateActivityItem/InstalledCertificateActivityItem.tsx

```
@@ -0,0 +1,39 @@
+      className={baseClass}
+      activity={activity}
+      hideCancel
+      hideShowDetails
+    >
+      {isFailed ? (
+        <>
+          <b>Fleet</b> failed to install certificate{" "}
+          <b>{activity.details?.certificate_name}</b> on this host.
+          {activity.details?.detail && (
+            <> Detail: {activity.details.detail}</>
+          )}
```

ESLint

[error] Replace `(⏎············<>·Detail:·{activity.details.detail}</>⏎··········)` with `<>·Detail:·{activity.details.detail}</>`

(prettier/prettier)